src/components/templates/UiQuestion/UiQuestion.stories.js

```diff
@@ -296,9 +297,13 @@ export const AsSimpleQuestion = {
       :button-issue-attrs="buttonIssueAttrs"
       :notification-feedback-attrs="notificationFeedbackAttrs"
     >
+    <template #title>
+      <UiHeading id="question-heading">{{ title }}</UiHeading>
+    </template>
```

Comment:
I talked to Piotr P. and we agreed that answers as a group will connected with title of the question